src/descriptors/power_system_structs.json

```diff
@@ -92,6 +92,67 @@
         ],
         "supertype": "OperationalCost"
     },
+    {
+        "struct_name": "PGLIBCost",
```

Comment:
This should MultiStageCost or something along those lines to generalize better

Reply:
I named it MultiStartCost & ThermalMultiStart, I know these are probably not the best names I could come up with but open to any suggestions